
    <html>
    <head>
    </head>
    <body>
      <div>
          <h2>Tree:</h2>
            <p>
            
                
                    .<br>
                
                    ├─■──nucleotide_pearson_blosum_nucleic_alignments ── Topic: 0<br>
                
                    └─■──agttggcatggtgccccaaattgggg_tgctcgta_aagcttaccaaaattgaagggacgttgacgtagggggacgctttag_aatctagcaattattga ── Topic: 1<br>
                
                    <br>
                
            
            </p>
      </div>

      <div>
        <h2>Index:</h2>

        <h3>Summaries</h3>
        
            <a href="#sum_2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf summary</a><br>
        

        <br>

        <h3>Topics and Docs</h3>
        
        <a href="#t_0">TOPIC 0: 	nucleotide, pearson, blosum</a><br>
            
                &emsp;<a href="#d_0">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_0, doc-id 0</a><br>
            
                &emsp;<a href="#d_2">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_2, doc-id 2</a><br>
            
                &emsp;<a href="#d_4">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_4, doc-id 4</a><br>
            
                &emsp;<a href="#d_6">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_6, doc-id 6</a><br>
            
                &emsp;<a href="#d_7">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_7, doc-id 7</a><br>
            
                &emsp;<a href="#d_8">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_8, doc-id 8</a><br>
            
                &emsp;<a href="#d_10">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_10, doc-id 10</a><br>
            
                &emsp;<a href="#d_11">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_11, doc-id 11</a><br>
            
                &emsp;<a href="#d_12">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 12</a><br>
            
                &emsp;<a href="#d_13">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 13</a><br>
            
                &emsp;<a href="#d_14">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 14</a><br>
            
                &emsp;<a href="#d_15">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 15</a><br>
            
                &emsp;<a href="#d_16">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 16</a><br>
            
                &emsp;<a href="#d_17">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 17</a><br>
            
                &emsp;<a href="#d_18">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 18</a><br>
            
                &emsp;<a href="#d_21">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_15, doc-id 21</a><br>
            
                &emsp;<a href="#d_22">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_16, doc-id 22</a><br>
            
                &emsp;<a href="#d_23">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_17, doc-id 23</a><br>
            
                &emsp;<a href="#d_24">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_18, doc-id 24</a><br>
            
                &emsp;<a href="#d_25">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_19, doc-id 25</a><br>
            
                &emsp;<a href="#d_26">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_20, doc-id 26</a><br>
            
                &emsp;<a href="#d_28">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 28</a><br>
            
                &emsp;<a href="#d_29">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 29</a><br>
            
                &emsp;<a href="#d_30">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 30</a><br>
            
                &emsp;<a href="#d_31">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 31</a><br>
            
                &emsp;<a href="#d_32">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 32</a><br>
            
                &emsp;<a href="#d_33">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 33</a><br>
            
                &emsp;<a href="#d_34">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 34</a><br>
            
                &emsp;<a href="#d_35">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 35</a><br>
            
                &emsp;<a href="#d_36">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 36</a><br>
            
                &emsp;<a href="#d_37">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 37</a><br>
            
                &emsp;<a href="#d_38">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 38</a><br>
            
                &emsp;<a href="#d_39">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 39</a><br>
            
                &emsp;<a href="#d_40">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 40</a><br>
            
                &emsp;<a href="#d_41">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 41</a><br>
            
                &emsp;<a href="#d_43">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_25, doc-id 43</a><br>
            
                &emsp;<a href="#d_44">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_26, doc-id 44</a><br>
            
                &emsp;<a href="#d_45">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_27, doc-id 45</a><br>
            
                &emsp;<a href="#d_46">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_28, doc-id 46</a><br>
            
                &emsp;<a href="#d_47">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_29, doc-id 47</a><br>
            
                &emsp;<a href="#d_48">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_30, doc-id 48</a><br>
            
                &emsp;<a href="#d_49">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_31, doc-id 49</a><br>
            
                &emsp;<a href="#d_50">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_32, doc-id 50</a><br>
            
                &emsp;<a href="#d_51">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_33, doc-id 51</a><br>
            
                &emsp;<a href="#d_53">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_35, doc-id 53</a><br>
            
                &emsp;<a href="#d_55">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_37, doc-id 55</a><br>
            
                &emsp;<a href="#d_59">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_41, doc-id 59</a><br>
            
                &emsp;<a href="#d_61">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_43, doc-id 61</a><br>
            
                &emsp;<a href="#d_63">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_45, doc-id 63</a><br>
            
                &emsp;<a href="#d_64">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_46, doc-id 64</a><br>
            
                &emsp;<a href="#d_65">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_47, doc-id 65</a><br>
            
                &emsp;<a href="#d_66">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_48, doc-id 66</a><br>
            
                &emsp;<a href="#d_67">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 67</a><br>
            
                &emsp;<a href="#d_68">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 68</a><br>
            
                &emsp;<a href="#d_69">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 69</a><br>
            
                &emsp;<a href="#d_70">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 70</a><br>
            
                &emsp;<a href="#d_71">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 71</a><br>
            
                &emsp;<a href="#d_72">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 72</a><br>
            
                &emsp;<a href="#d_73">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 73</a><br>
            
                &emsp;<a href="#d_74">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 74</a><br>
            
                &emsp;<a href="#d_75">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_50, doc-id 75</a><br>
            
                &emsp;<a href="#d_76">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_51, doc-id 76</a><br>
            
                &emsp;<a href="#d_77">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_52, doc-id 77</a><br>
            
                &emsp;<a href="#d_78">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_53, doc-id 78</a><br>
            
                &emsp;<a href="#d_79">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_54, doc-id 79</a><br>
            
                &emsp;<a href="#d_80">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_55, doc-id 80</a><br>
            
                &emsp;<a href="#d_81">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_56, doc-id 81</a><br>
            
                &emsp;<a href="#d_82">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_57, doc-id 82</a><br>
            
                &emsp;<a href="#d_83">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_58, doc-id 83</a><br>
            
                &emsp;<a href="#d_84">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_59, doc-id 84</a><br>
            
                &emsp;<a href="#d_85">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_60, doc-id 85</a><br>
            
                &emsp;<a href="#d_86">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_61, doc-id 86</a><br>
            
                &emsp;<a href="#d_87">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_62, doc-id 87</a><br>
            
                &emsp;<a href="#d_88">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_63, doc-id 88</a><br>
            
                &emsp;<a href="#d_89">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_64, doc-id 89</a><br>
            
                &emsp;<a href="#d_90">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_65, doc-id 90</a><br>
            
                &emsp;<a href="#d_91">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_66, doc-id 91</a><br>
            
                &emsp;<a href="#d_92">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_67, doc-id 92</a><br>
            
                &emsp;<a href="#d_93">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_68, doc-id 93</a><br>
            
                &emsp;<a href="#d_94">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_69, doc-id 94</a><br>
            
                &emsp;<a href="#d_95">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_70, doc-id 95</a><br>
            
                &emsp;<a href="#d_96">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_71, doc-id 96</a><br>
            
                &emsp;<a href="#d_97">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_72, doc-id 97</a><br>
            
                &emsp;<a href="#d_98">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_73, doc-id 98</a><br>
            
                &emsp;<a href="#d_99">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_74, doc-id 99</a><br>
            
                &emsp;<a href="#d_100">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_75, doc-id 100</a><br>
            
                &emsp;<a href="#d_101">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_76, doc-id 101</a><br>
            
                &emsp;<a href="#d_102">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_77, doc-id 102</a><br>
            
                &emsp;<a href="#d_103">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_78, doc-id 103</a><br>
            
                &emsp;<a href="#d_104">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_79, doc-id 104</a><br>
            
                &emsp;<a href="#d_105">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_80, doc-id 105</a><br>
            
                &emsp;<a href="#d_106">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_81, doc-id 106</a><br>
            
                &emsp;<a href="#d_107">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_82, doc-id 107</a><br>
            
                &emsp;<a href="#d_108">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_83, doc-id 108</a><br>
            
                &emsp;<a href="#d_109">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_84, doc-id 109</a><br>
            
                &emsp;<a href="#d_110">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_85, doc-id 110</a><br>
            
                &emsp;<a href="#d_111">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_86, doc-id 111</a><br>
            
                &emsp;<a href="#d_112">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 112</a><br>
            
                &emsp;<a href="#d_113">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 113</a><br>
            
                &emsp;<a href="#d_114">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 114</a><br>
            
                &emsp;<a href="#d_115">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 115</a><br>
            
                &emsp;<a href="#d_116">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 116</a><br>
            
                &emsp;<a href="#d_117">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_88, doc-id 117</a><br>
            
                &emsp;<a href="#d_118">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_89, doc-id 118</a><br>
            
                &emsp;<a href="#d_119">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_90, doc-id 119</a><br>
            
        
        <a href="#t_1">TOPIC 1: 	agttggcatg, tgctcgta, aagcttacca</a><br>
            
                &emsp;<a href="#d_1">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_1, doc-id 1</a><br>
            
                &emsp;<a href="#d_3">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_3, doc-id 3</a><br>
            
                &emsp;<a href="#d_5">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_5, doc-id 5</a><br>
            
                &emsp;<a href="#d_9">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_9, doc-id 9</a><br>
            
                &emsp;<a href="#d_19">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_13, doc-id 19</a><br>
            
                &emsp;<a href="#d_20">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_14, doc-id 20</a><br>
            
                &emsp;<a href="#d_27">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_21, doc-id 27</a><br>
            
                &emsp;<a href="#d_42">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_24, doc-id 42</a><br>
            
                &emsp;<a href="#d_52">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_34, doc-id 52</a><br>
            
                &emsp;<a href="#d_54">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_36, doc-id 54</a><br>
            
                &emsp;<a href="#d_56">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_38, doc-id 56</a><br>
            
                &emsp;<a href="#d_57">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_39, doc-id 57</a><br>
            
                &emsp;<a href="#d_58">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_40, doc-id 58</a><br>
            
                &emsp;<a href="#d_60">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_42, doc-id 60</a><br>
            
                &emsp;<a href="#d_62">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_44, doc-id 62</a><br>
            
        

      </div>

      <div>
        <h2>Summaries:</h2>
        
            <h3 id="sum_2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf</h3><br>
            
                <p> Bioinformatics I - H S 2023 lecture slides are partially based on an earlier Bioinfo1 lecture by Johannes Ftterer, ETH Z. Christian von Mering: Sequences in biology are produced in a defined direction and are usually also written in the same direction. Order of characters is crucial for sequence-function. Biological processes interconvert sequences N H2-Ser.Trp.His.Gly.Pro..-COO H S W H G A P: 20 different characters (+2): ACDEFGHIKLMNPQRSTVW Y (+ Se Cys (U); Pyr Lys(O)</p>
            
                <p> There are many possible definitions of similarity: length, character content, character distribution,.. Biological definition: (interrupted) stretches of identical or similar characters AATCTTACCAAAATTGAAGGGACGTTGACGTAGGGGGACGCTTTA G. Scoring systems are context independent (i.e. values are the same for every occurence of a given pair) They can be written in form of a simple matrix : Nucleotide identity matrix. Matrices show scoring values Mij for every conceivable pair i and j. Matrices are symmetrical Scoring Systems for calculation of similarity. Scores calculated for every possible alignment between two strings, ranked, and compared to random alignments for a statistical evaluation.</p>
            
                <p> PA M = Point Accepted Mutation (or percent accepted mutation) PA M1 reflects an evolutionary distance where 1% of characters have been changed. Alignment score = probability that the sequences confirm the working hypothesis, i.e. they are very closely related (1% difference) = product of matrix values for every pair in the alignment Matrix for greater divergence: PA M2 = 2% mismatched; PA M4 = 4% mismatchatched; etc. Actual values for DN A comparisons are usually derived differently. Scoring values depend on position in a DN A sequence.</p>
            
                <p> DN A comparison is only useful for closely related sequences or highly conserved motives DN A/DN A vs. protein/protein comparisons. Amino acids are differentially related to each other 1. required steps for mutation (distance): e.g. Thr codons (AC N) Ser (UC N) Cys (UG Y) for different purposes, amino acids may be sorted differently. Aminosum: BL Ock S Ubstitution Matrix derived from BLOCK S database (Henikoff and Henikoff, 1992) Multiple alignments of distantly related sequences (1764 blocks in 437 protein groups)</p>
            
                <p> Different matrices reflect different evolutionary relationships. Alignment matrix: Positions with locally positive alignment score are marked x Positions: characters words genes  Sequence alignment: algorithms  Dot Plot, where the best alignment could be (marked diagonal regions) Alignment Repeats Synteny of genes on mouse and human X chromosome. Detection of repeats within a sequence can be found in programs like Dotter http:// sonnhammer.sbc.su.se/Dotter.html Problem: Visual, no calculation of best alignment.</p>
            
                <p> BLAS T: Basic Local Alignment Search Tool (Altschul et al., 1990; 1997) FAS T: An algorithm that usually, but not always, works that gives nearly the right answer. Algorithms: Needleman-Wunsch (global) / Smith-Waterman (local) Highest possible alignment score guaranteed. But calculation-time and storage intensive: n x m to n m2 calculation steps required (n = m) for storage. Solution: Restrict search space by pre-selection of promising regions.</p>
            
                <p> BLAS T: Basic Local Alignment Search Tool. Search for ungapped regions sharing more than one exact k-tuple FAST A (Pearson and Lipmap, 1988) Speed is largely determined by ktup size for finding initial identities: the longer the faster, but with reduced sensitivity. BLAS T wuBLAS T fAST A: Multiple sequence Align DOTTE R Multiple seq Align Mult Alin. Blast in all flavours and all flavours of flavours. Blast: Search space in Gen Bank Gen Bank (NI H2/NI H) Blast is based on GC G package (UNI X/BLAS T)</p>
            
                <p> BLAS T programs  programs can be used with many different databases or subsections of databases. PS I BLAS T (position specific iterative BLAS T) uses a detected alignment to calculate a new PSS M. RP S-BLAS T Reverse Position-Specific BLAS T compares a protein query to predefined PSS Ms for known conserved protein domains. MegaBLAS T optimized for aligning longer sequences that differ only slightly uses longer words and a different algorithm (greedy algorithm) faster, works with longer sequences than BLAS T for nucleic acids (option in blastn) MegaB Last uses different type of words for initiation of alignments: words can be discontiguous, e.g.</p>
            
                <p> Multiple Sequence Alignment software is a guide to intelligent mutagenesis for functional analyses. Phylogeny reconstruction is based on gene/protein sequences. Structure prediction and tree construction are based on multiple alignments. Back to the page you came from: Peroxiredoxins are conserved markers of circadian rhythms. For more information on how to use the software, visit http://en.org/wiki/Sequence_alignment_software.com/multiple-sequence-alignment-software.</p>
            
                <p> PA M distance 3.3 (Human - Monkey) is the minimum. The likelihood is the probability of the data given the model. The aim of maximum likelihood is to choose the value of the parameter that maximizes the chance of finding the data. How do we verify a tree? Difficult! Very few trees are actually known with certainty with certainty a. Simulation b) Bootstrapping is the most difficult way to verify the tree with certainty, with the help of a simulation and bootstrapping.</p>
            
            <br>
        
      </div>

      <div>
      
        <div>
        <h2 id="t_0">TOPIC 0: 	nucleotide, pearson, blosum</h2>
            
                <div>
                    <h3 id="d_0">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_0, doc-id 0</h3>
                    <p style="margin:10px">
                        
                            Bioinformatics I -	 HS 2023<br>
                        
                            Lecture 1 (19.09.2023)<br>
                        
                            Biological Sequence Informatics<br>
                        
                            mering@mls.uzh.ch<br>
                        
                            Christian von Mering<br>
                        
                            the lecture slides are partially based on an earlier Bioinfo1 lecture by Johannes Fütterer, ETHZ<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_2">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_2, doc-id 2</h3>
                    <p style="margin:10px">
                        
                            Sequences in Molecular Biology<br>
                        
                            “Central Dogma”:                       DNA               RNA               Protein<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_4">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_4, doc-id 4</h3>
                    <p style="margin:10px">
                        
                            DNA<br>
                        
                            RNA<br>
                        
                            Protein<br>
                        
                            Metabolites, Cellular and Organismal Structures<br>
                        
                            Sequence<br>
                        
                            Information<br>
                        
                            Genes, Programs, Amplification<br>
                        
                            Production of Proteins; Structure<br>
                        
                            Structure (=&gt; Function)<br>
                        
                            Function<br>
                        
                            Stable, long-term storage of the  <br>
                        
                            whole information package<br>
                        
                            Transient amplification of specific  <br>
                        
                            parts; program execution<br>
                        
                            Program execution<br>
                        
                            Sequences contain information<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_6">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_6, doc-id 6</h3>
                    <p style="margin:10px">
                        
                            Short DNA pieces: assemble correctly<br>
                        
                            structure of genomes<br>
                        
                            Sequenced RNA: align with genome<br>
                        
                            RNA from special conditions<br>
                        
                            location and structure of genes  <br>
                        
                            expression control of genes<br>
                        
                            Sequenced Proteins: align with RNA<br>
                        
                            proteins from special conditions<br>
                        
                            structure of protein coding regions  <br>
                        
                            control of protein production  control <br>
                        
                            of function<br>
                        
                            posttranslational modifications <br>
                        
                            Comparison of individual sequences<br>
                        
                            deduction of function  <br>
                        
                            evolutionary relationships  <br>
                        
                            characterization of individual<br>
                        
                            Comparison of sequence collections<br>
                        
                            ecosystem characterization  <br>
                        
                            detection of presence of organism  <br>
                        
                            expression profiles<br>
                        
                            Required: Methods to compare sequence strings  <br>
                        
                            quantitatively to  determine their similarity <br>
                        
                            Sequence informatics – what can be learned ?<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_7">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_7, doc-id 7</h3>
                    <p style="margin:10px">
                        
                            Biological basis for sequence alignment<br>
                        
                            =&gt; many genes are related by common descent<br>
                        
                            Chymotrypsin<br>
                        
                            Trypsinogen<br>
                        
                            […]<br>
                        
                            […]<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_8">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_8, doc-id 8</h3>
                    <p style="margin:10px">
                        
                            Similarity: The degree to which two items share certain characters <br>
                        
                            Homology: descended from a shared common ancestor <br>
                        
                            Orthology:  derived from common ancestor during speciation <br>
                        
                            (often with retained function) <br>
                        
                            Paralogy:    evolved in parallel after gene duplication <br>
                        
                            (often with diverged function) <br>
                        
                            Note: Sequences are either homologous or not <br>
                        
                            Analogy: similarity without homology, e.g. due to convergent evolution <br>
                        
                            Sequence similarity: two sequences contain a number of identical or related characters in        <br>
                        
                            corresponding positions<br>
                        
                            Similarity of biological sequences: some definitions<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_10">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_10, doc-id 10</h3>
                    <p style="margin:10px">
                        
                            Chymotrypsin<br>
                        
                            Trypsinogen<br>
                        
                            […]<br>
                        
                            […]<br>
                        
                            A realistic example<br>
                        
                            -<br>
                        
                            Why was ”NG” aligned and not “QL”? <br>
                        
                            -<br>
                        
                            What does the “+” mean ? <br>
                        
                            -<br>
                        
                            How “good” is my alignment? <br>
                        
                            Why is it so difficult !?    Isn’t this trivial ?<br>
                        
                            =&gt; there are many possible ways to align two sequences !!<br>
                        
                            =&gt; we need a formalized scoring system, <br>
                        
                                  to describe and measure similarities, and to develop statistics … <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_11">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_11, doc-id 11</h3>
                    <p style="margin:10px">
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            5<br>
                        
                            0<br>
                        
                            0<br>
                        
                            0<br>
                        
                            G<br>
                        
                            0<br>
                        
                            5<br>
                        
                            0<br>
                        
                            0<br>
                        
                            T<br>
                        
                            0<br>
                        
                            0<br>
                        
                            5<br>
                        
                            0<br>
                        
                            C<br>
                        
                            0<br>
                        
                            0<br>
                        
                            0<br>
                        
                            5<br>
                        
                            General scoring systems are context independent (i.e. values are the same for every occurence of  a given <br>
                        
                            pair). They can be written in form of a simple matrix : <br>
                        
                            e.g. Nucleotide identity matrix  (positive score only for identities; no penalty for mismatch)<br>
                        
                            e.g. Nucleotide substitution matrix  (positive scores also for certain subtitutions)<br>
                        
                            The matrices show scoring values Mij for every conceivable pair i and j.  <br>
                        
                            Matrices are symmetrical<br>
                        
                            Scoring systems for calculation of “similarity”<br>
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            4<br>
                        
                            1<br>
                        
                            0<br>
                        
                            0<br>
                        
                            G<br>
                        
                            1<br>
                        
                            4<br>
                        
                            0<br>
                        
                            0<br>
                        
                            T<br>
                        
                            0<br>
                        
                            0<br>
                        
                            4<br>
                        
                            1<br>
                        
                            C<br>
                        
                            0<br>
                        
                            0<br>
                        
                            1<br>
                        
                            4<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_12">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 12</h3>
                    <p style="margin:10px">
                        
                            Quantification according to an Identity or Substitution Matrix <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_13">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 13</h3>
                    <p style="margin:10px">
                        
                            The matrix assigns a value to every possible character pair that could be observed in a comparison <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_14">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 14</h3>
                    <p style="margin:10px">
                        
                            In an alignment of two strings of characters, the scoring values for all occuring characterpairs are  <br>
                        
                            combined  to produce a score for the respective alignment <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_15">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 15</h3>
                    <p style="margin:10px">
                        
                            Scores are calculated for every possible alignment between two strings, ranked, and compared to random  <br>
                        
                            alignments for a statistical evaluation =&gt; Optimal Alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_16">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 16</h3>
                    <p style="margin:10px">
                        
                            Question: How can we derive realistic scoring values?<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_17">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 17</h3>
                    <p style="margin:10px">
                        
                            Rules for scoring values are based on a specific model for the origin of  <br>
                        
                            the expected similarity<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_18">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_12, doc-id 18</h3>
                    <p style="margin:10px">
                        
                            Scoring values for sequence alignments<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_21">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_15, doc-id 21</h3>
                    <p style="margin:10px">
                        
                            Biological sequences change by (almost) random mutation: <br>
                        
                            substitutions of characters	<br>
                        
                            (e.g. CG	<br>
                        
                            TG; Ala-Thr-Gly<br>
                        
                            Ala-Ser-Gly)<br>
                        
                            insertions<br>
                        
                            indels<br>
                        
                            (e.g.	 CTGG-ACAG<br>
                        
                            CTGGAACAG)<br>
                        
                            deletions<br>
                        
                            Most of the currently observed sequence variation has been fixed during evolution <br>
                        
                            Evolutionary fixed (“accepted”) sequence variation is restricted by functionality <br>
                        
                            =&gt; in functional sequences some mutations are deleterous: less likely to become fixed  <br>
                        
                            =&gt; in „non-functional“ sequences mutations are less consequential =&gt; more mutations fixed <br>
                        
                            ⇒ Even though every nucleotide and amino acid may technically mutate with almost equal frequency, in  real life <br>
                        
                            not all mutations in a coding sequence are observed with equal frequency. <br>
                        
                            ⇒   pij (probability that i to j change is accepted and can be observed) depends on i and j; <br>
                        
                            pij may serve as a quantitative measure of similarity between i and j <br>
                        
                            Rules for quantitative evaluation of changes in comparisons of biological sequences are based on  <br>
                        
                            theoretically or empirically derived “models” of evolution<br>
                        
                            }<br>
                        
                            Evolution as the basis of scoring systems<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_22">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_16, doc-id 22</h3>
                    <p style="margin:10px">
                        
                            Working hypothesis:<br>
                        
                            - the compared sequences are evolutionary related <br>
                        
                            - they differ by N% altered characters (mutations)<br>
                        
                            PAM = Point Accepted Mutation (or “percent accepted mutation”) <br>
                        
                            PAM1 reflects an evolutionary distance where 1% of characters have been changed: <br>
                        
                            =&gt; 99% of character pairs of an aligment should be identical; 1% mismatched <br>
                        
                            ⇒mutation matrix<br>
                        
                            Alignment score = probability that the sequences confirm the working hypothesis, i.e. they are very  closely <br>
                        
                            related (1% difference) = product of matrix values for every pair in the alignment <br>
                        
                            Matrix for greater divergence: <br>
                        
                            PAM2 = 2% mismatched;    PAM4 = 4% mismatched;   etc.<br>
                        
                            The PAM concept<br>
                        
                            Transitions = 3 x Transversions <br>
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            0.99<br>
                        
                            0.0033<br>
                        
                            0.0033<br>
                        
                            0.0033<br>
                        
                            G<br>
                        
                            0.0033<br>
                        
                            0.99<br>
                        
                            0.0033<br>
                        
                            0.0033<br>
                        
                            T<br>
                        
                            0.0033<br>
                        
                            0.0033<br>
                        
                            0.99<br>
                        
                            0.0033<br>
                        
                            C<br>
                        
                            0.0033<br>
                        
                            0.0033<br>
                        
                            0.0033<br>
                        
                            0.99<br>
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            0.99<br>
                        
                            0.006<br>
                        
                            0.002<br>
                        
                            0.002<br>
                        
                            G<br>
                        
                            0.006<br>
                        
                            0.99<br>
                        
                            0.002<br>
                        
                            0.002<br>
                        
                            T<br>
                        
                            0.002<br>
                        
                            0.002<br>
                        
                            0.99<br>
                        
                            0.006<br>
                        
                            C<br>
                        
                            0.002<br>
                        
                            0.002<br>
                        
                            0.006<br>
                        
                            0.99<br>
                        
                            Transitions (Pu-Pu/Py-Py changes) = Transversions (Pu-Py changes)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_23">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_17, doc-id 23</h3>
                    <p style="margin:10px">
                        
                            	<br>
                        
                             <br>
                        
                            Conversion of mutation matrix to log-odds matrix:  score <br>
                        
                            sij of a match between nucleotides i and j: <br>
                        
                            sij = log (piMij/pipj) = log (Mij/pj) (=log(observed frequency/expected frequency)) <br>
                        
                            Pi or j : frequency of nucleotide i or j (= 0.25); Mij: value from the mutation matrix <br>
                        
                            log base is only a scaling factor; frequently log2, values for matrix rounded to next integer<br>
                        
                            log-odds matrix: total alignment score is determined by addition of individual sij values  <br>
                        
                            (instead of multiplication of individual probability values) <br>
                        
                            Note: This is a convenient example for the PAM/Log-odds concept.  <br>
                        
                            Actual values for DNA comparisons are usually derived differently.<br>
                        
                            The log-odds concept<br>
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            2<br>
                        
                            -5<br>
                        
                            -7<br>
                        
                            -7<br>
                        
                            G<br>
                        
                            -2<br>
                        
                            2<br>
                        
                            -7<br>
                        
                            -7<br>
                        
                            T<br>
                        
                            -7<br>
                        
                            -7<br>
                        
                            2<br>
                        
                            -2<br>
                        
                            C<br>
                        
                            -7<br>
                        
                            -7<br>
                        
                            -2<br>
                        
                             2<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_24">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_18, doc-id 24</h3>
                    <p style="margin:10px">
                        
                            	<br>
                        
                             <br>
                        
                            Scoring  Values Depend on Position in a Sequence<br>
                        
                            Useful for finding specific  <br>
                        
                            functional sequence  <br>
                        
                            motives (TF binding sites,  <br>
                        
                            protein domains)<br>
                        
                            a quick aside: Position Specific Scoring Matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_25">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_19, doc-id 25</h3>
                    <p style="margin:10px">
                        
                            simple scoring systems treat all matches as equal and all mismatches as equal  a <br>
                        
                            negative score is assigned to mismatches <br>
                        
                            Alignments without gaps (only substitutions allowed):<br>
                        
                            seq 1 is closer related to seq 2 than to seq 3<br>
                        
                            	<br>
                        
                            But natural sequence variation also involves indels ! <br>
                        
                            =&gt; Gaps in sequence alignments must be possible<br>
                        
                            DNA comparison<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_26">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_20, doc-id 26</h3>
                    <p style="margin:10px">
                        
                            values for introduction of gaps are estimated empirically: <br>
                        
                            Affine gap: high value (A) for gap opening + lower value (B) for every gap elongation step <br>
                        
                            Gap penalty P for a gap of length n: P = A + nB<br>
                        
                            Alignments with gaps (indels allowed; increases possibilities drastically):<br>
                        
                            DNA comparison with gaps<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_28">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 28</h3>
                    <p style="margin:10px">
                        
                            Information content of a character in a string depends on the number of possible characters that could be  found at <br>
                        
                            that position. <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_29">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 29</h3>
                    <p style="margin:10px">
                        
                            Information content of (sub-)sequences containing only few or a subset of the possible characters is lower <br>
                        
                            e.g. AATAATTAAAATAAATAA for DNA (longer stretch of only 2 of the 4 possible characters)  or <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_30">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 30</h3>
                    <p style="margin:10px">
                        
                            LLDELDDELLDEL for protein (longer stretch of only 3 of the 20 possible characters) <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_31">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 31</h3>
                    <p style="margin:10px">
                        
                            In sequence comparison programs such «Sequences of low complexity” are sometimes filtered and  marked <br>
                        
                            as xxxxxxxx or in lower case. <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_32">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 32</h3>
                    <p style="margin:10px">
                        
                            The occurence of such sequences can be biologically relevant, but the sequences are not included in the  calculation of <br>
                        
                            similarity values, because character pairing would result in high scores even though it may occur  by chance (i.e. any <br>
                        
                            alignment of two such regions will produce a relative high score by chance and not  because the aligned positions are <br>
                        
                            homologous).<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_33">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_22, doc-id 33</h3>
                    <p style="margin:10px">
                        
                            Information content of biological sequences<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_34">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 34</h3>
                    <p style="margin:10px">
                        
                            Information content per position in DNA is low <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_35">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 35</h3>
                    <p style="margin:10px">
                        
                            only 4 possibilities =&gt; similarity by chance <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_36">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 36</h3>
                    <p style="margin:10px">
                        
                            DNA may contain many positions with no or small functional importance <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_37">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 37</h3>
                    <p style="margin:10px">
                        
                            e.g. non-coding regions, third codon positions  such <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_38">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 38</h3>
                    <p style="margin:10px">
                        
                            positions may not be conserved in evolution <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_39">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 39</h3>
                    <p style="margin:10px">
                        
                            =&gt; relatedness may be overlooked<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_40">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 40</h3>
                    <p style="margin:10px">
                        
                            DNA-DNA comparison is only useful<br>
                        
                            for closely related sequences  <br>
                        
                            or highly conserved motives<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_41">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_23, doc-id 41</h3>
                    <p style="margin:10px">
                        
                            DNA/DNA   vs. protein/protein comparisons<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_43">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_25, doc-id 43</h3>
                    <p style="margin:10px">
                        
                            	<br>
                        
                             <br>
                        
                            six possible reading phases per DNA sequence:<br>
                        
                            3 nucleotides form a codon =&gt; 64 codons possible <br>
                        
                            20 different amino acids in proteins <br>
                        
                            1 to 6 different codons per amino acid <br>
                        
                            =&gt; different DNA molecules can code for same protein<br>
                        
                            Conceptual translation of DNA into protein<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_44">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_26, doc-id 44</h3>
                    <p style="margin:10px">
                        
                            Model: Amino acids are differentially related to each other <br>
                        
                            1.	<br>
                        
                            required steps for mutation (“distance”): <br>
                        
                            e.g.  Thr codons (ACN)	<br>
                        
                            Ser (UCN) (one step) <br>
                        
                            Cys (UGY) (at least two steps)<br>
                        
                            2.<br>
                        
                            similar physico-chemical characteristics:<br>
                        
                            e.g.<br>
                        
                            aromatic side chains:  <br>
                        
                            basic side chains:<br>
                        
                            Phe, Tyr, Trp (F, Y, W) <br>
                        
                            Lys, Arg, His (K, R, H)<br>
                        
                            for different purposes, amino acids may be sorted differently <br>
                        
                            3.	<br>
                        
                            actually observed evolutionary conservation <br>
                        
                            Many different possibilities for reasonable substitution matrices<br>
                        
                            Amino acid substitution matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_45">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_27, doc-id 45</h3>
                    <p style="margin:10px">
                        
                            based on comparisons of evolutionary related proteins <br>
                        
                            PAM concept was developed for amino acid substitutions <br>
                        
                            Computed by Dayhoff (1978) based on a model of protein evolution <br>
                        
                            Model: Protein evolution through point mutations: <br>
                        
                            1) independent from previous substitutions <br>
                        
                            2) independent from the neighbouring amino acid <br>
                        
                            reality is more complex: e.g. 3-D and functional constraints  <br>
                        
                            Analysis of closely related sequences: <br>
                        
                            71 protein families <br>
                        
                            85% identity (allows for manual global alignment, few indels, minimal multiple changes)  in total <br>
                        
                            1572 accepted changes <br>
                        
                            ⇒Dayhoff matrix form for PAM n : <br>
                        
                            (log-odds matrix)<br>
                        
                            (a.a.= amino acid)<br>
                        
                            Popular amino acid substitution matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_46">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_28, doc-id 46</h3>
                    <p style="margin:10px">
                        
                            ---Ai--- <br>
                        
                            ---Aj---<br>
                        
                            Alignment:<br>
                        
                            Question: Do Ai and Aj align by chance or because they are evolutionary related ? <br>
                        
                            (i,j) value in PAM matrix gives the probability ratio of the two possibilities<br>
                        
                            PAM250<br>
                        
                            &gt;0: likely mutation <br>
                        
                            &lt;0: unlikely mutation<br>
                        
                            Note: Two sequences differing by 100 PAM units  do <br>
                        
                            not differ in all positions: <br>
                        
                            Effect of multiple substitutions at one site<br>
                        
                            PAM matrices<br>
                        
                            Important:  <br>
                        
                            in the protein world, spatial alignment of 3D <br>
                        
                            structures can act as “ground truth” of  <br>
                        
                            biologically meaningful alignment !<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_47">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_29, doc-id 47</h3>
                    <p style="margin:10px">
                        
                            Blosum: BLOck SUbstitution Matrix <br>
                        
                            derived from BLOCKS database (Henikoff and Henikoff, 1992) <br>
                        
                            multiple alignments of distantly related sequences (1764 blocks in 437 protein groups)<br>
                        
                            Aligned sequences<br>
                        
                            Conserved Blocks <br>
                        
                            First column in conserved block: AABACA: 6 AA pairs, 4 AB pairs, 4 AC pairs, 1 BC pair <br>
                        
                            =&gt; Observed probability of an AA pair: 6/15, AB pair 4/15 etc. <br>
                        
                            These values can be converted to a log-odds matrix <br>
                        
                            qij: normalized observed pair frequency between i and j <br>
                        
                            eij: normalized expected frequency of pair ij  (reflects the frequency of occurence of both characters)<br>
                        
                            BLOSUM matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_48">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_30, doc-id 48</h3>
                    <p style="margin:10px">
                        
                            BLOSUM 62: All sequences in the BLOCKS with &gt;62% similarity  were <br>
                        
                            collapsed in one. <br>
                        
                            Removes bias due to contribution of too many closely related  <br>
                        
                            sequences in the sequence collection.<br>
                        
                            Various BLOSUM matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_49">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_31, doc-id 49</h3>
                    <p style="margin:10px">
                        
                            PAM vs. BLOSUM matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_50">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_32, doc-id 50</h3>
                    <p style="margin:10px">
                        
                            All matrices work <br>
                        
                            Comparison of closely related sequences: low number PAM or high number BLOSUM matrices better  <br>
                        
                            Comparison of less related sequences: high number PAM or low number BLOSUM matrices <br>
                        
                            Routine: BLOSUM 62; if in doubt, try several different matrices<br>
                        
                            PAM vs. BLOSUM matrices<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_51">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_33, doc-id 51</h3>
                    <p style="margin:10px">
                        
                            For commonly used programs: <br>
                        
                            Matrix derived from frequency of actually occuring amino acid pairs in corresponding positions of  conserved <br>
                        
                            protein regions; values indicate probability that a given amino acid pair reflects evolutionary  relationship and <br>
                        
                            does not occur by chance. Different matrices reflect different evolutionary relationships.<br>
                        
                            Protein substitution matrix - summary<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_53">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_35, doc-id 53</h3>
                    <p style="margin:10px">
                        
                            DotPlot <br>
                        
                            =&gt;Rough idea, where the best alignment could be  <br>
                        
                            (marked diagonal regions) <br>
                        
                            Detection of repeats within a sequence <br>
                        
                            Identities can be characters, strings, genes…….<br>
                        
                            Performed by programs like “Dotter”  http://<br>
                        
                            sonnhammer.sbc.su.se/Dotter.html<br>
                        
                            Problem: <br>
                        
                            Visual, no calculation of the best alignment<br>
                        
                            Alignment<br>
                        
                            Repeats<br>
                        
                            Synteny of genes on mouse and  <br>
                        
                            human X chromosome<br>
                        
                            Dot Plots<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_55">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_37, doc-id 55</h3>
                    <p style="margin:10px">
                        
                            -<br>
                        
                            T<br>
                        
                            G<br>
                        
                            C<br>
                        
                            T<br>
                        
                            C<br>
                        
                            G<br>
                        
                            T<br>
                        
                            A<br>
                        
                            0	<br>
                        
                            - <br>
                        
                            1.<br>
                        
                            T <br>
                        
                            2.<br>
                        
                            T <br>
                        
                            3.<br>
                        
                            C <br>
                        
                            4.<br>
                        
                            A<br>
                        
                            5	<br>
                        
                            T<br>
                        
                            6	<br>
                        
                            A<br>
                        
                            Path matrix: Every node represents the endpoint of an  alignment, <br>
                        
                            i.e. all characters above and to the left are aligned.  Algorithms <br>
                        
                            calculate the optimal score for alignment to a node  and use this as <br>
                        
                            basis for calculation of all possible paths  starting from there.<br>
                        
                            Path Matrix<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_59">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_41, doc-id 59</h3>
                    <p style="margin:10px">
                        
                            Formalism<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_61">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_43, doc-id 61</h3>
                    <p style="margin:10px">
                        
                            Trace-back step: optimal alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_63">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_45, doc-id 63</h3>
                    <p style="margin:10px">
                        
                            Algorithms:  Needleman-Wunsch (global) / Smith-Waterman (local) <br>
                        
                            Highest possible alignment score guaranteed <br>
                        
                            But <br>
                        
                            calculation-time and storage intensive: <br>
                        
                            n x m to n x m2 calculation steps required (n &lt; m); n x m for storage<br>
                        
                            Too slow for database searches <br>
                        
                            Solution: restrict search space by pre-selection of “promising” regions<br>
                        
                            Optimal alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_64">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_46, doc-id 64</h3>
                    <p style="margin:10px">
                        
                            BLAST: Basic Local Alignment Search Tool  (Altschul et al., 1990; 1997) <br>
                        
                            FASTA: FAST-All (Lipman and Pearson, 1985; Pearson and Lipman, 1988) <br>
                        
                            Definition “Heuristic&#34;: An algorithm that usually, but not always, works or that gives nearly <br>
                        
                            the right  answer. <br>
                        
                            Principle: <br>
                        
                            Sequences with significant similarity contain short strings (words) with identity <br>
                        
                            1. Divide query in all possible words (1 to 4 for amino acids; 6 to 14 for DNA)  word <br>
                        
                            lengths: “k-tuples” <br>
                        
                            2. Determine positions of matching words in each database sequence <br>
                        
                            =&gt; hot-spots, hits <br>
                        
                            3. Attempt to extend hit-alignments in both directions without introduction of gaps <br>
                        
                            =&gt; high scoring segment pairs (hsp) <br>
                        
                            4. Extend alignments with introduction of gaps <br>
                        
                            Speed and sensitivity depend on search-parameters and on the choice of primary hits that will be  <br>
                        
                            processed further.<br>
                        
                            Faster sequence alignment: heuristics<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_65">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_47, doc-id 65</h3>
                    <p style="margin:10px">
                        
                            Step 1: define “promising” diagonals: search for ungapped regions sharing more than  one <br>
                        
                            exact k-tuple<br>
                        
                            FASTA (Pearson and Lipmap, 1988)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_66">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_48, doc-id 66</h3>
                    <p style="margin:10px">
                        
                            http://en.wikipedia.org/wiki/FASTA<br>
                        
                            speed is largely  determined <br>
                        
                            by ktup size for  finding the <br>
                        
                            initial identities:  the longer <br>
                        
                            the faster, but  with reduced <br>
                        
                            sensitivity<br>
                        
                            FASTA (Pearson and Lipmap, 1988)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_67">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 67</h3>
                    <p style="margin:10px">
                        
                            Altschul et al., 1990; 1997<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_68">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 68</h3>
                    <p style="margin:10px">
                        
                            Similar to FASTA, with some alterations <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_69">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 69</h3>
                    <p style="margin:10px">
                        
                            1. Define matching (not only identical) words with scores above a given threshold.  <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_70">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 70</h3>
                    <p style="margin:10px">
                        
                            Word size e.g. 11 for DNA  or 3 for proteins.  = hits <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_71">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 71</h3>
                    <p style="margin:10px">
                        
                            2. Search two hits within a predefined distance (e.g. &lt;40 amino acids) on a diagonal and  <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_72">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 72</h3>
                    <p style="margin:10px">
                        
                            combine them in a high scoring segment pair (HSP) <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_73">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 73</h3>
                    <p style="margin:10px">
                        
                            3. Initiate gapped extension (dynamic programming) only on the best HSP<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_74">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_49, doc-id 74</h3>
                    <p style="margin:10px">
                        
                            BLAST: Basic Local Alignment Search Tool<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_75">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_50, doc-id 75</h3>
                    <p style="margin:10px">
                        
                            BLAST words (neighborhood words)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_76">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_51, doc-id 76</h3>
                    <p style="margin:10px">
                        
                            matching words: <br>
                        
                            the sum of substitution values (derived from a scoring matrix) for a word pair must exceed a  <br>
                        
                            predefined threshold  (is often fixed in web-based applications) <br>
                        
                            successful hit-extension: <br>
                        
                            extension occurs as long as the new score does not drop more than a defined threshold below the  so far <br>
                        
                            obtained highest score <br>
                        
                            output: <br>
                        
                            longest alignment that cannot be improved by further elongation <br>
                        
                            theory of substitution score for ungapped alignments is well established  no <br>
                        
                            theory for introduction of gaps =&gt; empirical gap penalties <br>
                        
                            choice of gap penalties and substitution matrix influences output<br>
                        
                            BLAST sequence alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_77">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_52, doc-id 77</h3>
                    <p style="margin:10px">
                        
                            Bit score:  S’ = (λS- lnK)/ln2 <br>
                        
                            S: raw score <br>
                        
                            λ: log base of scoring matrix  <br>
                        
                            K: scale of search space size<br>
                        
                            Expectation value: E = mn2-S’ <br>
                        
                            m: length of query <br>
                        
                            n: length of database sequence<br>
                        
                            Expectation value: Frequency of an accidental alignment with the respective score in a given  search <br>
                        
                            procedure (=comparison of obtained score with scores of all other alignments obtained  in the search); <br>
                        
                            the smaller the better <br>
                        
                            Some other programs use<br>
                        
                            Z score: Z =<br>
                        
                                (score – average score of N permutations) 	<br>
                        
                             <br>
                        
                            standard deviation of randomized score distribution<br>
                        
                            Z score: compares actual score to score of N (e.g. 100) randomized sequences with the same  <br>
                        
                            character frequencies;  Z &gt; 3 often regarded as significant <br>
                        
                            (note: this is not the z score of FastA!)<br>
                        
                            score<br>
                        
                            Number of alignments<br>
                        
                            Scores follow an extreme  <br>
                        
                            value distribution<br>
                        
                            Significance of BLAST hits<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_78">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_53, doc-id 78</h3>
                    <p style="margin:10px">
                        
                            Alignment methods <br>
                        
                            BLAST  <br>
                        
                            wuBLAST  <br>
                        
                            FASTA  <br>
                        
                            LALIGN  <br>
                        
                            DOTTER <br>
                        
                            Multiple seq Align  <br>
                        
                            MultAlin<br>
                        
                            http://www.ncbi.nlm.nih.gov/; ftp://ncbi.nlm.nih.gov/blast  <br>
                        
                            http://www.ebi.ac.uk/  http://www.ebi.ac.uk/Tools/sss/<br>
                        
                            fasta/nucleotide.html  http://www.ch.embnet.org/software/<br>
                        
                            LALIGN_form.html  http://www.sanger.ac.uk/resources/<br>
                        
                            software/seqtools/  http://www.ebi.ac.uk/Tools/msa/  http://<br>
                        
                            multalin.toulouse.inra.fr/multalin/<br>
                        
                            Motifs and patterns <br>
                        
                            BLOCKS <br>
                        
                            Pfam <br>
                        
                            PROSITE (+many links)<br>
                        
                            http://blocks.fhcrc.org  http://www.sanger.ac.uk/resources/<br>
                        
                            databases/pfam.html  http://expasy.org<br>
                        
                            Presentation Methods <br>
                        
                            ALSCRIPT<br>
                        
                            http:// http://www.csb.yale.edu/userguides/seq/alscript/<br>
                        
                            Conversion utilities <br>
                        
                            http://www.ebi.ac.uk/Tools/sfc/ <br>
                        
                            Phylogenetic resources<br>
                        
                            (huge collection of links):<br>
                        
                            http://evolution.genetics.washington.edu/phylip/software.html<br>
                        
                            Search for Life Science Web services <br>
                        
                            http://www.biocatalogue.org/ <br>
                        
                            http://en.wikipedia.org/wiki/Sequence_alignment_software<br>
                        
                            Implementations and methods<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_79">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_54, doc-id 79</h3>
                    <p style="margin:10px">
                        
                            EBI: European Bioinformatics Institute (EMBL)  <br>
                        
                            http://www.ebi.ac.uk/ <br>
                        
                            FASTA3, wuBlast2 <br>
                        
                            NCBI: National Center for Biotechnology Information (NIH)  <br>
                        
                            http://www.ncbi.nlm.nih.gov/ <br>
                        
                            Blast in all flavours <br>
                        
                            Search space in GenBank Aug. 2015: 199’823’644’287 bases in 187’066’846 sequences <br>
                        
                            + 1’163’275’601’001 bases in 302’955’543 WGS records <br>
                        
                            Local sequence searches: <br>
                        
                            GCG package (UNIX based; FASTA and many other options)  Downloadable <br>
                        
                            stand-alone versions of new ncbiBLASTs and wuBLAST2<br>
                        
                            Websites for sequence searches<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_80">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_55, doc-id 80</h3>
                    <p style="margin:10px">
                        
                            Protein family<br>
                        
                            Smith-W.<br>
                        
                            oriBLAST<br>
                        
                            BLAST<br>
                        
                            PSI-BLAST<br>
                        
                            Serine Protease<br>
                        
                            275<br>
                        
                            273<br>
                        
                            275<br>
                        
                            286<br>
                        
                            Ras<br>
                        
                            255<br>
                        
                            249<br>
                        
                            252<br>
                        
                            375<br>
                        
                            Globin<br>
                        
                            28<br>
                        
                            26<br>
                        
                            28<br>
                        
                            623<br>
                        
                            Cytochrome P450<br>
                        
                            211<br>
                        
                            197<br>
                        
                            211<br>
                        
                            224<br>
                        
                            run time<br>
                        
                            36<br>
                        
                            1.0<br>
                        
                            0.34<br>
                        
                            0.87<br>
                        
                            Altschul et al., Nucleic Acids Res. 25, 3389-3402 (1997)<br>
                        
                            Comparison of search performance<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_81">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_56, doc-id 81</h3>
                    <p style="margin:10px">
                        
                            General substitution matrices are build on collections of „all proteins“ <br>
                        
                            In functional sequence elements or protein motives the variation of some (or all) sequence  <br>
                        
                            positions may be restricted by functional constraints. <br>
                        
                            This can be modelled more specifically by <br>
                        
                            substitution matrices build only on a specific motif  <br>
                        
                            Hidden Markov Models (HMMs) for a specific model  <br>
                        
                            Regular expressions <br>
                        
                            e.g. PSI BLAST (Position specific iterative BLAST) uses a detected alignment to calculate a  new <br>
                        
                            PSSM (position specific scoring matrix) and performs a new BLAST with this PSSM etc.  More <br>
                        
                            distant relationships may be detected. <br>
                        
                            Precalculated PSSMs for known protein domains form the basis of „Conserved domain“ CD  <br>
                        
                            search, automatically performed whenever protein sequences are submitted to BLAST<br>
                        
                            Use of ambiguous words, pattern profiles<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_82">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_57, doc-id 82</h3>
                    <p style="margin:10px">
                        
                            programs<br>
                        
                            Database<br>
                        
                            Query<br>
                        
                            Comments<br>
                        
                            blastp<br>
                        
                            protein<br>
                        
                            protein<br>
                        
                            finds also distant relationships<br>
                        
                            blastn<br>
                        
                            nucleotide<br>
                        
                            nucleotide<br>
                        
                            default for close relationships<br>
                        
                            blastx<br>
                        
                            protein<br>
                        
                            translated  <br>
                        
                            nucleotide<br>
                        
                            useful for analysis of new DNA  and <br>
                        
                            EST sequences<br>
                        
                            tblastn<br>
                        
                            translated  <br>
                        
                            nucleotide<br>
                        
                            protein<br>
                        
                            unannotated coding regions in  <br>
                        
                            database sequences<br>
                        
                            tblastx<br>
                        
                            translated  <br>
                        
                            nucleotide<br>
                        
                            translated  <br>
                        
                            nucleotide<br>
                        
                            EST analysis<br>
                        
                            BLAST programs<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_83">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_58, doc-id 83</h3>
                    <p style="margin:10px">
                        
                            programs<br>
                        
                            Comments<br>
                        
                            BLAST1.4<br>
                        
                            first BLAST version<br>
                        
                            QBLAST =BLAST2.0<br>
                        
                            current NCBI default; <br>
                        
                            “2 hit search strategy” for increased speed; performs  <br>
                        
                            “gapped BLAST”<br>
                        
                            PSI-BLAST<br>
                        
                            Position-Specific Iterative BLAST: generates a PSSM from  <br>
                        
                            multiple alignments of a protein query to a database and  uses <br>
                        
                            the PSSM repeatedly to search for more distant hits  PSSM = <br>
                        
                            position-specific scoring matrix (option in blastp)<br>
                        
                            PHI-BLAST<br>
                        
                            Pattern Hit Initiated BLAST: seeks for alignments that  <br>
                        
                            preserve a specific protein motif (option in blastp)<br>
                        
                            RPS-BLAST<br>
                        
                            Reverse Position-Specific BLAST: compares a protein  <br>
                        
                            query to predefined PSSMs for known conserved  protein <br>
                        
                            domains. Invoked by activating CD search in  the BLAST <br>
                        
                            window<br>
                        
                            Align 2 sequences<br>
                        
                            pairwise alignment of two defined sequences (now also incorporated as  <br>
                        
                            option in different BLAST programs)<br>
                        
                            Taxonomy BLAST<br>
                        
                            lists BLAST hits according to taxonomy<br>
                        
                            Special BLAST programs<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_84">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_59, doc-id 84</h3>
                    <p style="margin:10px">
                        
                            programs<br>
                        
                            Comments<br>
                        
                            MegaBLAST<br>
                        
                            optimized for aligning longer sequences that differ only slightly  uses <br>
                        
                            longer words and a different algorithm (“greedy algorithm”)  faster, <br>
                        
                            works with longer sequences than BLAST <br>
                        
                            for nucleic acids (option in blastn)<br>
                        
                            discontiguous MegaBLAST<br>
                        
                            uses a different type of words for initiation of alignments: <br>
                        
                            words can be discontiguous, e.g. 11 or 12 matches in a template  <br>
                        
                            region of 16, 18 or 21 nucleotides. <br>
                        
                            Options for 1 or 2 initial hits implemented <br>
                        
                            different possibilities to analyze coding and non-coding regions  <br>
                        
                            (differentiated by the importance of the third codon position)  for <br>
                        
                            nucleic acids (option in blastn)<br>
                        
                            BLAST to find short, almost perfect matches	<br>
                        
                            now many BLAST procedures recognize query length and <br>
                        
                            adapt search parameters automatically (option can be deactivated) <br>
                        
                            BLAST programs can be used with many different databases or subsections of databases<br>
                        
                            Special BLAST programs<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_85">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_60, doc-id 85</h3>
                    <p style="margin:10px">
                        
                            gi|1172846|sp|Q09028|RB48_HUMAN  CHROMATIN ASSEMBLY FACTOR...<br>
                        
                            Score	 E-value  <br>
                        
                            (bits) <br>
                        
                            131	<br>
                        
                            4e-31<br>
                        
                            Contains information of type and quality of matches<br>
                        
                            Score: raw score is calculated according to the number and weight of matches and gaps  <br>
                        
                            depends on substitution matrix used <br>
                        
                            bits score is normalized with parameters reflecting the search strategy <br>
                        
                            E-value: statistic significance of hit; describes roughly how often a hit with a given score can be  <br>
                        
                            expected to occur randomly with the applied search strategy <br>
                        
                            with short queries no low E-values can be expected!!<br>
                        
                            Output<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_86">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_61, doc-id 86</h3>
                    <p style="margin:10px">
                        
                            Page continues<br>
                        
                            	<br>
                        
                             <br>
                        
                            BLAST output <br>
                        
                            Number of detected hits (=  <br>
                        
                            similar sequences in the  <br>
                        
                            database) <br>
                        
                            Query sequence (=Input) <br>
                        
                            Regions where subject  <br>
                        
                            sequences (from the  <br>
                        
                            database) are similar (color  <br>
                        
                            code for alignment scores)<br>
                        
                            Output<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_87">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_62, doc-id 87</h3>
                    <p style="margin:10px">
                        
                            BLAST output<br>
                        
                            List of individual sequences  <br>
                        
                            from the database with  <br>
                        
                            similarity<br>
                        
                            E values and scores  (Max <br>
                        
                            score is for the best <br>
                        
                            aligned region between query  <br>
                        
                            and subject; if more than one  <br>
                        
                            region can be aligned or the  <br>
                        
                            same region several times,  the <br>
                        
                            Total score will be the<br>
                        
                            Note: the database may  <br>
                        
                            contain multiple entries of the  <br>
                        
                            same sequence (also with  <br>
                        
                            different names)<br>
                        
                            Output<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_88">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_63, doc-id 88</h3>
                    <p style="margin:10px">
                        
                            Actual sequence alignment: <br>
                        
                            Protein sequence in single letter code  of <br>
                        
                            query sequence (with gaps) and <br>
                        
                            of detected similar sequence <br>
                        
                            Identities between the two sequences or  <br>
                        
                            similarities with a positive alignment  score <br>
                        
                            (+)<br>
                        
                            Next hit (represents in this case two  <br>
                        
                            identical database entries) <br>
                        
                            Information about the quality  <br>
                        
                            Scores and E values like here are <br>
                        
                            probably not indicative for a significant  <br>
                        
                            homology (unless they come from  short, <br>
                        
                            very good alignment regions)<br>
                        
                            Output<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_89">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_64, doc-id 89</h3>
                    <p style="margin:10px">
                        
                            Data usage	<br>
                        
                             <br>
                        
                            not all protein sequences will yield helpful results  not <br>
                        
                            all “significant” matches will be found <br>
                        
                            not all found matches must be “significant” <br>
                        
                            relevant matches: <br>
                        
                            Prediction of protein function <br>
                        
                            Definition of functionally significant sequence motives in proteins or nucleic acids  <br>
                        
                            Determination of phylogenetic relationships (multiple alignments, tree construction)  <br>
                        
                            Structure prediction <br>
                        
                            Acquisition of complete sequences from partial sequence information (EST, peptide, etc.)  Guide <br>
                        
                            to intelligent mutagenesis for functional analyses <br>
                        
                            Assembly of complete genomes from partial sequences<br>
                        
                            Output interpretation<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_90">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_65, doc-id 90</h3>
                    <p style="margin:10px">
                        
                            substitution matrices <br>
                        
                            Gonnet, G.H., Cohen, M.A., and Benner, S.A. (1992) Exhaustive matching of the entire protein sequence  <br>
                        
                            database. Science 256, 1443–1145. <br>
                        
                            Henikoff, S. and Henikoff, J.G. (1992) Amino acid substitution matrices from protein blocks. Proceedings of  <br>
                        
                            the National Academy of Sciences U.S.A. 89, 10915–10919. <br>
                        
                            Schwartz, R.M. and Dayhoff, M.O. (1979) Matrices for detecting distant relationships, in Atlas of Protein  <br>
                        
                            Sequences and Structure (Dayhoff, M.O. ed.) 5, National Biomedical Research Foundation, Washington,  <br>
                        
                            D.C., U.S.A. pp. 353–358. <br>
                        
                            alignment programs <br>
                        
                            Altschul, S.F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990) Basic Local Alignment Tool.  <br>
                        
                            Journal of Molecular Biology 215, 403–410. <br>
                        
                            Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D.J. (1997) Gapped  <br>
                        
                            BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Research <br>
                        
                            25, 3389–3402. <br>
                        
                            Needleman, S.B. and Wunsch, C.D. (1970) A general method applicable to the search for similarities in the  <br>
                        
                            amino acid sequence of two proteins. Journal of Molecular Biology 48, 443–453. <br>
                        
                            Pearson, W.B. (1998) Empirical statistical estimates for sequence similarity searches. Journal of Molecular  <br>
                        
                            Biology 276, 71–84. <br>
                        
                            Pearson, W.R. and Lipman, D.J. (1988) Improved tools for biological sequence analysis. Proceedings of the  <br>
                        
                            National Academy of Sciences U.S.A. 85, 2444–2448. <br>
                        
                            Smith, T.F. and Waterman, M.S. (1981) Comparison of bio-sequences. Advances in Applied Mathematics 2,  <br>
                        
                            482–489. <br>
                        
                            Wilbur, W.J. and Lipman, D.J. (1983) Rapid Similarity Searches of Nucleic Acid and Protein Data Banks.  <br>
                        
                            Proceedings of the National Academy of Sciences U.S.A. 80, 726–730. <br>
                        
                            http://en.wikipedia.org/wiki/Sequence_alignment_software<br>
                        
                            References<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_91">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_66, doc-id 91</h3>
                    <p style="margin:10px">
                        
                            [Bonus Material: Multiple Sequence Alignment]<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_92">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_67, doc-id 92</h3>
                    <p style="margin:10px">
                        
                            Multiple Sequence Alignment <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_93">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_68, doc-id 93</h3>
                    <p style="margin:10px">
                        
                            From: Peroxiredoxins are conserved markers of circadian rhythms. Nature 485, 459–464 (24 May 2012) <br>
                        
                            Motivations for sequence alignment<br>
                        
                            1) to identify and check the state of “active sites”<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_94">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_69, doc-id 94</h3>
                    <p style="margin:10px">
                        
                            Motivations for sequence alignment<br>
                        
                            2) to identify and characterize “protein domains”<br>
                        
                            Pyruvate Kinase<br>
                        
                            “PH”-domain (pink); <br>
                        
                            occurring in two different proteins<br>
                        
                            PH<br>
                        
                            PH<br>
                        
                            definition: <br>
                        
                            “parts of proteins that can  <br>
                        
                            evolve, function, and exist <br>
                        
                            independently of the rest” <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_95">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_70, doc-id 95</h3>
                    <p style="margin:10px">
                        
                            Motivations for sequence alignment<br>
                        
                            3) to make phylogenetic inferrences (“trees”)<br>
                        
                            canine 1 <br>
                        
                            canine 2 <br>
                        
                            canine 3 <br>
                        
                            […] <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_96">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_71, doc-id 96</h3>
                    <p style="margin:10px">
                        
                            Multiple Alignment<br>
                        
                            Combinatorial Explosion:    very many possible solutions <br>
                        
                            Complexity: O(alignment_length                  ) <br>
                        
                            =&gt; an NP-complete problem  !!<br>
                        
                            number_seqs<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_97">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_72, doc-id 97</h3>
                    <p style="margin:10px">
                        
                            Multiple Alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_98">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_73, doc-id 98</h3>
                    <p style="margin:10px">
                        
                            Quality of MSA: Benchmarking<br>
                        
                            Structural Aligments <br>
                        
                            offer the best  <br>
                        
                            benchmarks !<br>
                        
                            “BAliBASE”: <br>
                        
                            Benchmark Alignment Database<br>
                        
                            Hand-made multiple sequence alignments <br>
                        
                            Based on selected structural alignments<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_99">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_74, doc-id 99</h3>
                    <p style="margin:10px">
                        
                            Phylogeny  <br>
                        
                            Reconstruction <br>
                        
                            - a quick overview -<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_100">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_75, doc-id 100</h3>
                    <p style="margin:10px">
                        
                            Some iconic phylogenetic trees<br>
                        
                            Charles Darwin, personal notebook<br>
                        
                            HIV (AIDS),  <br>
                        
                            and closely related <br>
                        
                            animal viruses.<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_101">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_76, doc-id 101</h3>
                    <p style="margin:10px">
                        
                            Some iconic phylogenetic trees<br>
                        
                            current, conventional version of the ‘tree of life’<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_102">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_77, doc-id 102</h3>
                    <p style="margin:10px">
                        
                            an alternative view,  <br>
                        
                            emphasizing some problems<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_103">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_78, doc-id 103</h3>
                    <p style="margin:10px">
                        
                            Generating phylogenetic trees <br>
                        
                            - from gene/protein sequences -<br>
                        
                            • Phenetic:  trees are constructed based on <br>
                        
                            observed characteristics directly, not on <br>
                        
                            evolutionary history <br>
                        
                            • Cladistic: trees are constructed based on <br>
                        
                            fitting observed characteristics to some <br>
                        
                            model of evolutionary history<br>
                        
                            Distance <br>
                        
                            methods<br>
                        
                            Parsimony <br>
                        
                            and <br>
                        
                            Maximum <br>
                        
                            Likelihood <br>
                        
                            methods<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_104">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_79, doc-id 104</h3>
                    <p style="margin:10px">
                        
                            B<br>
                        
                            A<br>
                        
                            C<br>
                        
                            D<br>
                        
                                Unrooted tree<br>
                        
                            M    Rooted tree       UnRooted Tree <br>
                        
                                   (2m-3)! / 2m-2(m-2)!    (2m-5)! / 2m-3(m-3)! 	<br>
                        
                            2.                     1                          1 <br>
                        
                            3                        3                          1 <br>
                        
                            4                      15                          3 <br>
                        
                            5                    105                        15 <br>
                        
                            6                    945                      105 <br>
                        
                            7                 10395                     945 <br>
                        
                            8               135135                 10395 <br>
                        
                            9             2027025               135135 <br>
                        
                            10          34459425            2027025 <br>
                        
                              Numer of topologies for m taxa<br>
                        
                                        Rooted Tree<br>
                        
                            O<br>
                        
                            A<br>
                        
                            B<br>
                        
                            C<br>
                        
                            D<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_105">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_80, doc-id 105</h3>
                    <p style="margin:10px">
                        
                            Which genes to use ?<br>
                        
                            Ribosomes <br>
                        
                            (RNA or proteins)<br>
                        
                            Polymerases<br>
                        
                            But, for recent events: <br>
                        
                            fast-evolving genes<br>
                        
                            … should occur in every organism <br>
                        
                            … should rarely undergo horizontal transfer <br>
                        
                            … should be evolving ‘slowly’ <br>
                        
                            … should only occur in one copy per genome <br>
                        
                            … should function in a process that sees no change<br>
                        
                            suitable marker genes …<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_106">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_81, doc-id 106</h3>
                    <p style="margin:10px">
                        
                            Example for a phenetic technique: Neighbor Joining<br>
                        
                            2) Distance matrix<br>
                        
                            1) Alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_107">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_82, doc-id 107</h3>
                    <p style="margin:10px">
                        
                            PAM distance 3.3 (Human - Monkey) is the minimum. So we&#39;ll <br>
                        
                            join Human and Monkey to MonHum and we&#39;ll calculate the new <br>
                        
                            distances.<br>
                        
                            Mon-Hum<br>
                        
                            Monkey<br>
                        
                            Human<br>
                        
                            Spinach<br>
                        
                            Mosquito<br>
                        
                            Rice<br>
                        
                            First Step<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_108">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_83, doc-id 108</h3>
                    <p style="margin:10px">
                        
                            After we have joined two species in a subtree we have to compute the <br>
                        
                            distances from every other node to the new subtree. We do this with a simple <br>
                        
                            average of distances: <br>
                        
                            Dist[Spinach, MonHum]  <br>
                        
                            	<br>
                        
                            = (Dist[Spinach, Monkey] + Dist[Spinach, Human])/2  <br>
                        
                            	<br>
                        
                            = (90.8 + 86.3)/2 = 88.55 <br>
                        
                            Mon-Hum<br>
                        
                            Monkey<br>
                        
                            Human<br>
                        
                            Spinach<br>
                        
                            Calculation of new distances<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_109">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_84, doc-id 109</h3>
                    <p style="margin:10px">
                        
                            Human<br>
                        
                            Mosquito<br>
                        
                            Mon-Hum<br>
                        
                            Monkey<br>
                        
                            Spinach<br>
                        
                            Rice<br>
                        
                            Mos-(Mon-Hum)<br>
                        
                            Next cycle<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_110">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_85, doc-id 110</h3>
                    <p style="margin:10px">
                        
                            Human<br>
                        
                            Mosquito<br>
                        
                            Mon-Hum<br>
                        
                            Monkey<br>
                        
                            Spinach<br>
                        
                            Rice<br>
                        
                            Mos-(Mon-Hum)<br>
                        
                            Spin-Rice<br>
                        
                            (Spin-Rice)-(Mos-(Mon-Hum))<br>
                        
                            Last joining<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_111">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_86, doc-id 111</h3>
                    <p style="margin:10px">
                        
                            • The likelihood is the probability of the data given <br>
                        
                            the model <br>
                        
                            • The probability of observing the data under the <br>
                        
                            assumed model will change depending on the <br>
                        
                            parameter values of the model.  <br>
                        
                            • The aim of maximum likelihood is to choose the <br>
                        
                            value of the parameter that maximizes the <br>
                        
                            probability of finding the data.  <br>
                        
                            Example for a cladistic technique: Maximum Likelihood<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_112">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 112</h3>
                    <p style="margin:10px">
                        
                            What is an evolutionary model in this context ?<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_113">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 113</h3>
                    <p style="margin:10px">
                        
                            Dayhoff matrix (Dayhoff et al., 1978) <br>
                        
                            JTT matrix (Jones et al., 1992) <br>
                        
                            mtREV matrix (Adachi and Hasegawa, 1996) <br>
                        
                            WAG matrix (Whelan and Goldman, 2001). <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_114">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 114</h3>
                    <p style="margin:10px">
                        
                            “an empirical matrix describing the relative rates of <br>
                        
                            amino acid replacements”<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_115">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 115</h3>
                    <p style="margin:10px">
                        
                            Typically, the model has additional free ‘parameters’: <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_116">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_87, doc-id 116</h3>
                    <p style="margin:10px">
                        
                            -The rate of evolution can vary across parts of the tree <br>
                        
                            -The rate of evolution can vary from site to site in the protein <br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_117">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_88, doc-id 117</h3>
                    <p style="margin:10px">
                        
                            How is maximum likelihood computed ?<br>
                        
                            T T A G<br>
                        
                            AT<br>
                        
                            GC<br>
                        
                            AT<br>
                        
                            GC<br>
                        
                            AT<br>
                        
                            GC<br>
                        
                            1) Image all ancestral possibilities <br>
                        
                            and evolutionary paths.<br>
                        
                            T T A G<br>
                        
                            G<br>
                        
                            T<br>
                        
                            G<br>
                        
                            L(path) = L(root) x Π L(branches)  <br>
                        
                            	<br>
                        
                            =P(GT)P(GG) P(GA)P(GG) […] <br>
                        
                                         <br>
                        
                            2) Compute the likelihood of each path<br>
                        
                            3) multiply all likelihoods  <br>
                        
                                over all possible paths<br>
                        
                            4) throughout, do not forget to  <br>
                        
                                 optimize all free parameters<br>
                        
                            5) Repeat for each tree  <br>
                        
                                 topology, identify the one  <br>
                        
                                 with best Likelihood<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_118">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_89, doc-id 118</h3>
                    <p style="margin:10px">
                        
                            How do we verify a tree?<br>
                        
                            Difficult !    Very few trees are actually known with certainty<br>
                        
                            a) Simulation <br>
                        
                            b) Bootstrapping<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_119">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_90, doc-id 119</h3>
                    <p style="margin:10px">
                        
                            <br>
                        
                    </p>
                </div>
            
        </div>
    
        <div>
        <h2 id="t_1">TOPIC 1: 	agttggcatg, tgctcgta, aagcttacca</h2>
            
                <div>
                    <h3 id="d_1">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_1, doc-id 1</h3>
                    <p style="margin:10px">
                        
                            DNA<br>
                        
                            RNA<br>
                        
                            Protein<br>
                        
                            Molecule<br>
                        
                            Sequence<br>
                        
                            5‘-AGTTGGCATGGTGCCCCAAATTGGGG-3‘ <br>
                        
                            3‘-TCAACCGTACCACGGGGTTTAACCCC-5‘ <br>
                        
                            4 different characters: ACGT<br>
                        
                            5‘-AGUUGGCAUGGUGCCCCAAAUUGGGG-3‘ <br>
                        
                            4 different characters: ACGU<br>
                        
                            NH2-Ser.Trp.His.Gly.Ala.Pro..-COOH   <br>
                        
                                 S	<br>
                        
                            W	<br>
                        
                            H	<br>
                        
                            G	<br>
                        
                            A	<br>
                        
                            P <br>
                        
                            20 different characters (+2): ACDEFGHIKLMNPQRSTVWY <br>
                        
                            (+ SeCys (U); PyrLys(O))<br>
                        
                            Sequences in Molecular Biology<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_3">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_3, doc-id 3</h3>
                    <p style="margin:10px">
                        
                            DNA<br>
                        
                            RNA<br>
                        
                            Protein<br>
                        
                            Process<br>
                        
                            Sequence<br>
                        
                            Note: Sequences in biology are produced in a defined direction and <br>
                        
                            are  usually also written in the same defined direction <br>
                        
                            Order of characters is crucial for sequence-function  AGTTG ≠ GTTGA <br>
                        
                            or VPQ ≠ QPV<br>
                        
                            5‘-AGTTGGCATGGTGCCCCAAATTGGGG-3‘ <br>
                        
                            3‘-TCAACCGTACCACGGGGTTTAACCCC-5‘<br>
                        
                            Replication<br>
                        
                            Reverse  <br>
                        
                            Transcription<br>
                        
                            RNA dep. Transcr. <br>
                        
                            Replication<br>
                        
                            5‘-AGUUGGCAUGGUGCCCCAAAUUGGGG-3‘<br>
                        
                            Transcription<br>
                        
                            5‘-CCCCAAUUUGGGGCACCAUGCCAACU-3‘<br>
                        
                            Biological processes interconvert sequences<br>
                        
                            NH2-Ser.Trp.His.Gly.Ala.Pro..-COOH <br>
                        
                                 S   W   H   G   A   P <br>
                        
                                  V   G   M   . . . <br>
                        
                                   L   A   W   . . . (depending on start point)<br>
                        
                            Translation<br>
                        
                            or<br>
                        
                            or<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_5">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_5, doc-id 5</h3>
                    <p style="margin:10px">
                        
                            DNA<br>
                        
                            RNA<br>
                        
                            Protein<br>
                        
                            5‘-AGTTGGCATGGTGCCCCAAATTGGGG-3‘ <br>
                        
                            3‘-TCAACCGTACCACGGGGTTTAACCCC-5‘<br>
                        
                            NH2-Ser.Trp.His.Gly.Ala.Pro..-COOH  S	<br>
                        
                            W	<br>
                        
                            H	<br>
                        
                            G	<br>
                        
                            A	<br>
                        
                            P<br>
                        
                            5‘-AGUUGGCAUGGUGCCCCAAAUUGGGG-3‘ <br>
                        
                            5‘-CCCCAAUUUGGGGCACCAUGCCAACU-3‘<br>
                        
                            straightforward<br>
                        
                            5‘-AGU.UGG.CAU.GGU.GCC.CCA.AAU.UGG.GG-3‘ <br>
                        
                            --C.---.--C.--C.--U.--G.--C.---.-- <br>
                        
                            UC-.---.---.--A.--A.--C.---.---.-- <br>
                        
                            UCC.---.---.--G.--G.--U.---.---.-- <br>
                        
                            UCA.---.---.---.---.---.---.---.-- <br>
                        
                            UCG.---.---.---.---.---.---.---.--<br>
                        
                            ambiguous<br>
                        
                            RNA/DNA<br>
                        
                            Sequences can be interconverted computationally<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_9">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_9, doc-id 9</h3>
                    <p style="margin:10px">
                        
                            Challenge: Find the best possible alignment<br>
                        
                            E.g. search identical sequence segments for assembly of long sequences from short, overlapping fragments <br>
                        
                            AAGCTTACCAAAATTGAAGGGACGTTGACGTAGGGGGACGCTTTAG <br>
                        
                            GACGCTTTAGTTTAGCCACCGGTATTTAGC <br>
                        
                            Similar characters: physico-chemical characteristics, functional characteristics, evolutionary relation……  <br>
                        
                            Comparison of two (or more) sequences:  Alignment of identical and similar sequence segments <br>
                        
                            AAGCTTACCAAAATTGAAGGGACGTTGACGTAGGGGGACGCTTTAG <br>
                        
                            AATCTAGCAATTATTGAAGGGACGTTGACGAAGGGGTTCGCTACCG<br>
                        
                            Many possible definitions of “similarity”: length, character content, character distribution,…..<br>
                        
                            Biological definition: (interrupted) stretches of  identical or similar characters<br>
                        
                            AAGCTTACCAAAATTGAAGGGACGTTGACGTAGGGGGACGCTTTAG  <br>
                        
                            AATCTAGCAATTATTGAAGGGACGTTGACGAAGGGGTTCGCTACCG<br>
                        
                            (and do it fast)<br>
                        
                            Sequence Similarity<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_19">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_13, doc-id 19</h3>
                    <p style="margin:10px">
                        
                            Value in nucleotide identity matrix  = probability that characters in a pair are identical<br>
                        
                            Alignment score  =  probability that the aligned sequences are identical  = <br>
                        
                            Product of matrix values of each position of the alignment <br>
                        
                            Example: <br>
                        
                            AAGCTTACCAAAATTGAAGGGACGTTGACGTAGGGGGACGCTTTAG  <br>
                        
                            AATCTAGCAATTATTGAAGGGACGTTGACGAAGGGGTTCGCTACCG <br>
                        
                            Score of the global alignment = 0 <br>
                        
                            Score of local alignment (red substring): = 1 <br>
                        
                            Matrix produces a yes/no answer; ok for identity, not useful for similarity<br>
                        
                            The meaning of scoring values -  (I) probability<br>
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            1<br>
                        
                            0<br>
                        
                            0<br>
                        
                            0<br>
                        
                            G<br>
                        
                            0<br>
                        
                            1<br>
                        
                            0<br>
                        
                            0<br>
                        
                            T<br>
                        
                            0<br>
                        
                            0<br>
                        
                            1<br>
                        
                            0<br>
                        
                            C<br>
                        
                            0<br>
                        
                            0<br>
                        
                            0<br>
                        
                            1<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_20">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_14, doc-id 20</h3>
                    <p style="margin:10px">
                        
                            AAGCTTACCAAAATTGAAGGGACGTTGACGTAGGGGGACGCTTTAG  <br>
                        
                            AATCTAGCAATTATTGAAGGGACGTTGACGAAGGGGTTCGCTACCG<br>
                        
                            Global aligment = 34 matches of 46 pairs  <br>
                        
                            Red substring = 18 matches of 18 pairs<br>
                        
                            =&gt; Score 34 <br>
                        
                            =&gt; Score 18<br>
                        
                            Without normalization, the score will grow with the length of the alignment<br>
                        
                            Alignment score  =  sum of matrix values of each position of the alignment<br>
                        
                            Value in nucleotide identity matrix  =  arbitrary score for a matching pair<br>
                        
                            A<br>
                        
                            G<br>
                        
                            T<br>
                        
                            C<br>
                        
                            A<br>
                        
                            1<br>
                        
                            0<br>
                        
                            0<br>
                        
                            0<br>
                        
                            G<br>
                        
                            0<br>
                        
                            1<br>
                        
                            0<br>
                        
                            0<br>
                        
                            T<br>
                        
                            0<br>
                        
                            0<br>
                        
                            1<br>
                        
                            0<br>
                        
                            C<br>
                        
                            0<br>
                        
                            0<br>
                        
                            0<br>
                        
                            1<br>
                        
                            The meaning of scoring values -  (iI) arbitrary score<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_27">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_21, doc-id 27</h3>
                    <p style="margin:10px">
                        
                            Parameters<br>
                        
                            Smith-Waterman<br>
                        
                            FASTA<br>
                        
                            wuBLAST<br>
                        
                            ncbiBLAST<br>
                        
                            Match<br>
                        
                            5<br>
                        
                            5<br>
                        
                            5<br>
                        
                            1<br>
                        
                            Mismatch<br>
                        
                            -4<br>
                        
                            -4<br>
                        
                            -4<br>
                        
                            -3<br>
                        
                            Gap opening<br>
                        
                            -16<br>
                        
                            -16<br>
                        
                            -10<br>
                        
                            -5<br>
                        
                            Gap extension<br>
                        
                            -4<br>
                        
                            -4<br>
                        
                            -10<br>
                        
                            -2<br>
                        
                            AGATCAACGGATTGCTTTCCTGCCGCCATT  <br>
                        
                            AGATCTTCGGATT---TTCCTGGGGCCATT<br>
                        
                            S-W/F<br>
                        
                            wuBLAST ncbiBLAST<br>
                        
                            75<br>
                        
                            69<br>
                        
                            2<br>
                        
                            =&gt; scores can be only compared within one scoring system<br>
                        
                            “Real life” scoring parameters for DNA alignments<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_42">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_24, doc-id 42</h3>
                    <p style="margin:10px">
                        
                            Seq 1:	<br>
                        
                            CCTGGAGTCCAGCAAAAACGTC <br>
                        
                            Seq 2:	<br>
                        
                            CATGGTGACCACCGAAAAGCTC<br>
                        
                            15/22<br>
                        
                            Seq 3:	<br>
                        
                            GTTAGAAAGTTCTAAGAATGTG<br>
                        
                            9/22<br>
                        
                            Seq 2 seems to be much closer related to Seq 1<br>
                        
                            But:  Sequences have coding potential  for peptide sequences:<br>
                        
                            Seq 1:	<br>
                        
                            C.CTG.GAG.TCC.AGC.AAA.AAC.GTC <br>
                        
                            .leu.glu.ser.ser.lys.asn.val<br>
                        
                            Seq 2:	<br>
                        
                            C.ATG.GTG.ACC.ACC.GAA.AAG.CTC<br>
                        
                            15/22<br>
                        
                            .met.val.thr.thr.glu.lys.leu<br>
                        
                            0/7<br>
                        
                            Seq 3:	<br>
                        
                            G.TTA.GAA.AGT.TCT.AAG.AAT.GTG<br>
                        
                            9/22<br>
                        
                            .leu.glu.ser.ser.lys.asn.val<br>
                        
                            7/7<br>
                        
                            The peptide encoded by Seq 3 is identical to that encoded by Seq 1 ! <br>
                        
                            =&gt; always search at the level that carries the biological function<br>
                        
                            DNA/DNA   vs. protein/protein comparisons<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_52">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_34, doc-id 52</h3>
                    <p style="margin:10px">
                        
                            T C G G C T T A A C C G T T G A C A<br>
                        
                            A C G G A C T T T A C C G A T G C T T<br>
                        
                            - - - - - - x x x - - - - - x - - x x<br>
                        
                            - x - - - x - - - - x x - - - - x - -<br>
                        
                            - - x x - - - - - - - - x - - x - - -<br>
                        
                            - - x x - - - - - - - - x - - x - - -<br>
                        
                            - x - - - x - - - - x x - - - - x - -<br>
                        
                            - - - - - - x x x - - - - - x - - x x<br>
                        
                            - - - - - - x x x - - - - - x - - x x<br>
                        
                            x - - - x - - - - x - - - x - - - - -<br>
                        
                            x - - - x - - - - x - - - x - - - - -<br>
                        
                            - x - - - x - - - - x x - - - - x - -<br>
                        
                            - x - - - x - - - - x x - - - - x - -<br>
                        
                            - - x x - - - - - - - - x - - x - - -<br>
                        
                            - - - - - - x x x - - - - - x - - x x<br>
                        
                            - - - - - - x x x - - - - - x - - x x<br>
                        
                            - - x x - - - - - - - - x - - x - - -<br>
                        
                            x - - - x - - - - x - - - x - - - - -<br>
                        
                            - x - - - x - - - - x x - - - - x - -<br>
                        
                            x - - - x - - - - x - - - x - - - - -<br>
                        
                            Alignment matrix: <br>
                        
                            Positions with locally positive  <br>
                        
                            alignment score are marked x <br>
                        
                            “Positions”: <br>
                        
                            characters  <br>
                        
                            words  <br>
                        
                            genes <br>
                        
                            ……<br>
                        
                            Sequence alignment: algorithms<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_54">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_36, doc-id 54</h3>
                    <p style="margin:10px">
                        
                            T C G G C T T A A C C G T T G A C A<br>
                        
                            - <br>
                        
                            - <br>
                        
                            - <br>
                        
                            - <br>
                        
                            - <br>
                        
                            - <br>
                        
                            x <br>
                        
                            x<br>
                        
                            x - - - x - - <br>
                        
                            - x x - - - - <br>
                        
                            - x x - - - - <br>
                        
                            x - - - x - - <br>
                        
                            - - - - - x x <br>
                        
                            - - - - - x x <br>
                        
                            - - - x - - - <br>
                        
                            - - - x - - -<br>
                        
                            - x - - - x - - <br>
                        
                            - x - - - x - - <br>
                        
                            - - x x - - - - <br>
                        
                            - - - - - - x x <br>
                        
                            - - - - - - x x <br>
                        
                            - - x x - - - - <br>
                        
                            x - - - x - - - <br>
                        
                            - x - - - x - - <br>
                        
                            x - - - x - - -<br>
                        
                            A C G G A C T T T A C C G A T G C T T <br>
                        
                            - - - - - - x x x - - - - - x - - x x <br>
                        
                            - - x x - - - - x - - <br>
                        
                            - - - - x - - x - - - <br>
                        
                            - - - - x - - x - - - <br>
                        
                            - - x x - - - - x - - <br>
                        
                            x - - - - - x - - x x <br>
                        
                            x - - - - - x - - x x <br>
                        
                            - x - - - x - - - - - <br>
                        
                            - x - - - x - - - - - <br>
                        
                            - - x x - - - - x - - <br>
                        
                            - - x x - - - - x - - <br>
                        
                            - - - - x - - x - - - <br>
                        
                            x - - - - - x - - x x <br>
                        
                            x - - - - - x - - x x <br>
                        
                            - - - - x - - x - - - <br>
                        
                            - x - - - x - - - - - <br>
                        
                            - - x x - - - - x - - <br>
                        
                            - x - - - x - - - - -<br>
                        
                            Exploration of all paths through the alignment-matrix to identify the maximum value for  the <br>
                        
                            scoring function	<br>
                        
                            =&gt; optimal alignment guaranteed (global or local)<br>
                        
                            Many possible paths<br>
                        
                            dynamic programming: <br>
                        
                            an algorithmic technique in which an  <br>
                        
                            optimization problem is solved by caching  <br>
                        
                            subproblem solutions instead of  <br>
                        
                            recalculating them<br>
                        
                            Smith-Waterman algorithm  <br>
                        
                            (Needleman-Wunsch)<br>
                        
                            Searching for the optimal alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_56">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_38, doc-id 56</h3>
                    <p style="margin:10px">
                        
                            1<br>
                        
                            2<br>
                        
                            3<br>
                        
                            4<br>
                        
                            5<br>
                        
                            Node corresponding to T - T pair with all previous characters also paired <br>
                        
                            -	<br>
                        
                            T	<br>
                        
                            G	<br>
                        
                            C	<br>
                        
                            T	<br>
                        
                            C	<br>
                        
                            G	<br>
                        
                            T	<br>
                        
                            A <br>
                        
                            0<br>
                        
                            - <br>
                        
                            T  <br>
                        
                            T  <br>
                        
                            C  <br>
                        
                            A  <br>
                        
                            T  <br>
                        
                            A<br>
                        
                            -<br>
                        
                            TT5<br>
                        
                            TG 3<br>
                        
                            CC<br>
                        
                            8<br>
                        
                            AT 6<br>
                        
                            TC 4<br>
                        
                            AG <br>
                        
                            2<br>
                        
                            -TGCTCGTA <br>
                        
                            -TTCATA<br>
                        
                            Score = 2<br>
                        
                              <br>
                        
                            Score = Sum of  <br>
                        
                            alignment scores  <br>
                        
                            up to this position<br>
                        
                            Score function  <br>
                        
                            uses weights  <br>
                        
                            for matches,  <br>
                        
                            mismatches  <br>
                        
                            and gaps: <br>
                        
                            e.g. <br>
                        
                            Match = +5 <br>
                        
                            Mismatch = -2 <br>
                        
                            Gap = -6<br>
                        
                            Filling the Path Matrix (arbitrary choice)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_57">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_39, doc-id 57</h3>
                    <p style="margin:10px">
                        
                            -<br>
                        
                            T<br>
                        
                            G<br>
                        
                            C<br>
                        
                            T<br>
                        
                            C<br>
                        
                            G<br>
                        
                            T<br>
                        
                            A<br>
                        
                            0 <br>
                        
                            1<br>
                        
                            2<br>
                        
                            3<br>
                        
                            4 <br>
                        
                            5 <br>
                        
                            6<br>
                        
                            Match = +5 <br>
                        
                            Mismatch = -2 <br>
                        
                            Gap = -6<br>
                        
                            - <br>
                        
                            T  <br>
                        
                            T  <br>
                        
                            C  <br>
                        
                            A  <br>
                        
                            T  <br>
                        
                            A<br>
                        
                            -TGCTCGTA <br>
                        
                            ----TTCATA<br>
                        
                            -TGCTCGTA <br>
                        
                            -TTCATA<br>
                        
                            ---TGCTCGTA ---TGCTCGTA <br>
                        
                            -TTCA-TA	<br>
                        
                            -TTCA--TA<br>
                        
                            Filling the Path Matrix (by many ways)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_58">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_40, doc-id 58</h3>
                    <p style="margin:10px">
                        
                            0<br>
                        
                            -6	<br>
                        
                            -12	<br>
                        
                            -18	<br>
                        
                            -24	<br>
                        
                            -30	<br>
                        
                            -36	<br>
                        
                            -42	<br>
                        
                            -48<br>
                        
                            -6<br>
                        
                            -12<br>
                        
                            -30<br>
                        
                            -36<br>
                        
                            -<br>
                        
                            T<br>
                        
                            G<br>
                        
                            C<br>
                        
                            T<br>
                        
                            C<br>
                        
                            G<br>
                        
                            T<br>
                        
                            A<br>
                        
                            0	<br>
                        
                            -<br>
                        
                            1	<br>
                        
                            T<br>
                        
                            2	<br>
                        
                            T<br>
                        
                            3	<br>
                        
                            C	 -18<br>
                        
                            4	<br>
                        
                            A	 -24<br>
                        
                            5	<br>
                        
                            T<br>
                        
                            6	<br>
                        
                            A<br>
                        
                            Match = +5 <br>
                        
                            Mismatch = -2 <br>
                        
                            Gap = -6<br>
                        
                            5 <br>
                        
                            -12<br>
                        
                            TGCTCGTA  <br>
                        
                            TTCATA<br>
                        
                            -TGCTCGTA  <br>
                        
                            T-TCATA<br>
                        
                            or<br>
                        
                            -TTCATA<br>
                        
                            or T-GCTCGTA<br>
                        
                            Filling the Path Matrix (optimal)<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_60">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_42, doc-id 60</h3>
                    <p style="margin:10px">
                        
                            0<br>
                        
                            1<br>
                        
                            2<br>
                        
                            3<br>
                        
                            4<br>
                        
                            5<br>
                        
                            6<br>
                        
                            7<br>
                        
                            8<br>
                        
                            -<br>
                        
                            T<br>
                        
                            G<br>
                        
                            C<br>
                        
                            T<br>
                        
                            C<br>
                        
                            G<br>
                        
                            T<br>
                        
                            A<br>
                        
                            sequence B<br>
                        
                            Match = +5 <br>
                        
                            Mismatch = -2 <br>
                        
                            Gap = -6<br>
                        
                            0	<br>
                        
                            - <br>
                        
                            1.<br>
                        
                            T <br>
                        
                            2.<br>
                        
                            2.<br>
                        
                            T <br>
                        
                            3.<br>
                        
                            3.<br>
                        
                            C <br>
                        
                            4.<br>
                        
                            4.<br>
                        
                            A <br>
                        
                            5.<br>
                        
                            5.<br>
                        
                            T <br>
                        
                            6.<br>
                        
                            6.<br>
                        
                            A<br>
                        
                            0<br>
                        
                            -6<br>
                        
                            -12<br>
                        
                            -18<br>
                        
                            -24<br>
                        
                            -30<br>
                        
                            -36<br>
                        
                            -42<br>
                        
                            -48<br>
                        
                            -6<br>
                        
                            5<br>
                        
                            -1<br>
                        
                            -7<br>
                        
                            -13<br>
                        
                            -19<br>
                        
                            -25<br>
                        
                            -31<br>
                        
                            -37<br>
                        
                            -12<br>
                        
                            -1<br>
                        
                            3<br>
                        
                            -3<br>
                        
                            -2<br>
                        
                            -8<br>
                        
                            -14<br>
                        
                            -20<br>
                        
                            -26<br>
                        
                            -18<br>
                        
                            -7<br>
                        
                            -3<br>
                        
                            8<br>
                        
                            2<br>
                        
                            3<br>
                        
                            -3<br>
                        
                            -9<br>
                        
                            -15<br>
                        
                            -24<br>
                        
                            -13<br>
                        
                            -9<br>
                        
                            2<br>
                        
                            6<br>
                        
                            0<br>
                        
                            1<br>
                        
                            -5<br>
                        
                            -4<br>
                        
                            -30<br>
                        
                            -19<br>
                        
                            -15<br>
                        
                            -4<br>
                        
                            7<br>
                        
                            4<br>
                        
                            -2<br>
                        
                            6<br>
                        
                            0<br>
                        
                            -36<br>
                        
                            -25<br>
                        
                            -21<br>
                        
                            -10<br>
                        
                            1<br>
                        
                            5<br>
                        
                            2<br>
                        
                            0<br>
                        
                            11<br>
                        
                            Fill-in path matrix: calculate and store highest score for each <br>
                        
                            node,  Also store pointer to node from which the stored score was <br>
                        
                            calculated<br>
                        
                            sequence A<br>
                        
                            Filling the entire Path Matrix<br>
                        
                            <br>
                        
                    </p>
                </div>
            
                <div>
                    <h3 id="d_62">2023-09-19___Christian-von-Mering__Biological-Sequence-Informatics__UZH-BIO390-HS23-lecture-01.pdf /page_44, doc-id 62</h3>
                    <p style="margin:10px">
                        
                            0<br>
                        
                            -6	<br>
                        
                            -12	<br>
                        
                            -18	<br>
                        
                            -24	<br>
                        
                            -30	<br>
                        
                            -36	<br>
                        
                            -42	<br>
                        
                            -48<br>
                        
                            -6<br>
                        
                            5<br>
                        
                            -1	<br>
                        
                            -7	<br>
                        
                            -13	<br>
                        
                            -19	<br>
                        
                            -25	<br>
                        
                            -31	<br>
                        
                            -37<br>
                        
                            -12	<br>
                        
                            -1<br>
                        
                            3<br>
                        
                            -3	<br>
                        
                            -2	<br>
                        
                            -8	<br>
                        
                            -14	<br>
                        
                            -20	<br>
                        
                            -26<br>
                        
                            -18	<br>
                        
                            -7	<br>
                        
                            -3<br>
                        
                            8<br>
                        
                            2<br>
                        
                            3<br>
                        
                            -3	<br>
                        
                            -9	<br>
                        
                            -15<br>
                        
                            2<br>
                        
                            6<br>
                        
                            0<br>
                        
                            1<br>
                        
                            -5	<br>
                        
                            -4<br>
                        
                            -30	<br>
                        
                            -19	<br>
                        
                            -15	<br>
                        
                            -4<br>
                        
                            7<br>
                        
                            4<br>
                        
                            -2<br>
                        
                            6<br>
                        
                            0<br>
                        
                            -10	<br>
                        
                            1<br>
                        
                            5<br>
                        
                            2<br>
                        
                            0<br>
                        
                            11<br>
                        
                            0 <br>
                        
                            -<br>
                        
                            1 <br>
                        
                            T<br>
                        
                            2 <br>
                        
                            G<br>
                        
                            3 <br>
                        
                            C<br>
                        
                            4 <br>
                        
                            T<br>
                        
                            5 <br>
                        
                            C<br>
                        
                            6 <br>
                        
                            G<br>
                        
                            7 <br>
                        
                            T<br>
                        
                            8 <br>
                        
                            A<br>
                        
                            0	<br>
                        
                            -<br>
                        
                            1	<br>
                        
                            T<br>
                        
                            2	<br>
                        
                            T<br>
                        
                            3	<br>
                        
                            C<br>
                        
                            4	<br>
                        
                            A	 -24	<br>
                        
                            -13	<br>
                        
                            -9<br>
                        
                            5	<br>
                        
                            T<br>
                        
                            6	<br>
                        
                            A<br>
                        
                            sequence B<br>
                        
                            Match = +5 <br>
                        
                            Mismatch = -2 <br>
                        
                            Gap = -6<br>
                        
                            -36	<br>
                        
                            -25	<br>
                        
                            -21 <br>
                        
                            TGCTCGTA<br>
                        
                            T--TCATA<br>
                        
                            Trace-back path matrix: the optimal path may not produce the highest score at every step<br>
                        
                            TGC-TCGTA<br>
                        
                            TTCATA---<br>
                        
                            sequence A<br>
                        
                            Trace-back step: suboptimal alignment<br>
                        
                            <br>
                        
                    </p>
                </div>
            
        </div>
    
    </div>
    </body>
    </html>
    